readme.md

```diff
@@ -69,6 +69,7 @@
 - [pgMagic](https://pgmagic.app) - macOS app that lets you query your PostgreSQL database using natural language. ($)
 - [Atua](https://atua.app) - macOS app to use ChatGPT in any app. ($)
 - [16x Prompt](https://prompt.16x.engineer) - Cross-platform app to add source code and formatting instructions into prompt for coding tasks. ($)
+- [AI Chat](https://www.chatbar.top/) - Cross-platform desktop app that provides quick access to chatbots like ChatGPT from the menu bar.
```

Comment:
It only seems to provide access to ChatGPT, so the description is not correct.

Reply:
Plan to support Claude and Gemini in a month. This description avoid the need to update it weeks later. Is that OK? If not, I could correct it now.

Reply:
Then I'll leave this pull request open until that is done.

Reply:
Hi, the feature promised is released; and the linked web site is also updated to reflect that. Could you review this PR again?